Roommatch/Roommatch/ViewControllers/CreateProfileViewController.m

```diff
@@ -0,0 +1,156 @@
+//
+//  CreateProfileViewController.m
+//  Roommatch
+//
+//  Created by Lily Yang on 7/8/22.
+//
+
+#import "CreateProfileViewController.h"
+#import "Lib.h"
+#import "User.h"
+#import <Parse/Parse.h>
+
+@interface CreateProfileViewController () <UINavigationControllerDelegate,UIImagePickerControllerDelegate, UITextViewDelegate>
+
+@property (weak, nonatomic) IBOutlet UIImageView *imageView;
+
+@property (weak, nonatomic) IBOutlet UITextField *nameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *ageTextField;
+@property (weak, nonatomic) IBOutlet UITextField *pronounsTextField;
+@property (weak, nonatomic) IBOutlet UITextField *cityTextField;
+@property (weak, nonatomic) IBOutlet UITextView *bioTextView;
+@property (weak, nonatomic) IBOutlet UITextField *priceLow;
+@property (weak, nonatomic) IBOutlet UITextField *priceHigh;
+
+@property (weak, nonatomic) IBOutlet UILabel *charactersRemainingLabel;
+
+@property (weak, nonatomic) IBOutlet UISegmentedControl *smokingSegmentedControl;
+@property (weak, nonatomic) IBOutlet UISegmentedControl *petsSegmentedControl;
+@property (weak, nonatomic) IBOutlet UISegmentedControl *inCollegeSegmentedControl;
+
+@property (weak, nonatomic) IBOutlet UITextField *collegeNameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *instagramTagTextField;
+
+@property (nonatomic) BOOL choseImage;
+@property int charLimit;
+
+@end
+
+@implementation CreateProfileViewController
+
+- (void)viewDidLoad {
+    self.charLimit = 280; 
```

Comment:
It's good practice to put constants like this in static variables at the top of the file. This makes them easy to find. [Example](https://www.internalfb.com/code/fbsource/[c9a32a312ac7]/fbobjc/Apps/Instagram/AppLibraries/IGActivityPluggableModule/IGActivity/ViewControllers/Exported/IGActivityFeedViewController.mm?lines=261-266).